orchestra/static/orchestra/project/js/controllers.js

```diff
@@ -0,0 +1,870 @@
+(function () {
+  // 'use strict';
+
+  angular
+    .module('orchestra.project.controllers')
+    .controller('ProjectController', ProjectController);
+
+  ProjectController.$inject = ['$location', '$scope', '$routeParams', '$http', '$sce',
+                            '$compile', '$modal', '$timeout', 'dataService', 'orchestraApi'];
+
+  function ProjectController($location, $scope, $routeParams, $http, $sce,
+                          $compile, $modal, $timeout, dataService, orchestraApi) {
+    // var vm = this;
+    vm = $scope;
+    $scope.activate = function() {
+      // Prevent initial animation of vis load
+      d3.select('.project-management').style('opacity', 0);
```

Comment:
see my next comment about this, to see why i suggest this, but: encapsulate all d3 calls inside the *Vis functions.  that will allow you to have an angular controller/file without any implementation details, and you'll be able to put the vis in a different non-angular module/file